frontend/app/component/form/polling_station_choice/PollingStationChoiceForm.test.tsx

```diff
@@ -67,6 +71,7 @@ describe("Test PollingStationChoiceForm", () => {
   });
 
   test("Polling station list", async () => {
+    overrideOnce("get", "/api/polling_stations/1", 200, pollingStationMock);
```

Comment:
Testing the list endpoint should not have an overrideOnce on the detail endpoint. It should be on "/api/elections/{election_id}/polling_stations/". See also the other instances in this file

Reply:
I think I'll fix this by rebasing main and then change it. Thanks!